packages/widgetbook/lib/src/knobs/slider_knob.dart

```diff
@@ -5,17 +5,13 @@ import 'package:widgetbook/src/knobs/nullable_checkbox.dart';
 
 class SliderKnob extends Knob<double> {
   SliderKnob({
-    required String label,
-    String? description,
-    required double value,
+    required super.label,
+    super.description,
+    required super.value,
     this.max = 1,
     this.min = 0,
     this.divisions,
-  }) : super(
-          label: label,
-          description: description,
-          value: value,
-        );
+  });
```

Comment:
This should not be part of this PR, but we can leave it in as it improves the code 👍🏻